static/sass/_snapcraft_l-application.scss

```diff
@@ -22,7 +9,7 @@
       }
 
       .logo-text {
-        color: #fff;
+        color: var(--vf-color-text-default);
```

Comment:
The CSS variables are kind of Vanilla internals, and were not meant for direct external consumption originally.

While technically there is nothing wrong with such approach, it was meant to be used via SCSS variables (that are processed into CSS variables on build time).
The list of all theme variables is in Vanilla color definitions: https://github.com/canonical/vanilla-framework/blob/main/scss/_settings_colors.scss#L237-L293